russian/prayer/exmachina/couple/couple.md

```diff
@@ -0,0 +1,4 @@
+Подсесть то ты подсел, но ты же вчера гульбенил по-дикому... Забыл?!
+Омбрэ от твоих вчерашних посиделок настолько сильно что
+[Пара не выдерживает и уходит](leave/leave.md)
+[Ты сам не выдерживаешь и падаешь от приступа](disease/disease.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.